CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 ### Improvements
 
+* (all) [#23445](https://github.com/cosmos/cosmos-sdk/pull/23445) Remove `v2` code from codebase.
```

Comment:
this should go under [Unreleased] since this change won't be included in the [v0.52.0-rc.2] tag it's currently under

Reply:
or wait, is that not a live tag yet?

Reply:
im confused

Reply:
yeah this tag does not exist yet